core/src/main/java/nu/yona/server/analysis/entities/IntervalActivity.java

```diff
@@ -0,0 +1,53 @@
+package nu.yona.server.analysis.entities;
+
+import java.time.ZonedDateTime;
+import java.util.Date;
+import java.util.UUID;
+
+import javax.persistence.Entity;
+
+import nu.yona.server.entities.EntityWithID;
+import nu.yona.server.goals.entities.Goal;
+
+@Entity
+public abstract class IntervalActivity extends EntityWithID
+{
+	private UUID goalID;
+	private UUID userAnonymizedID;
+	private ZonedDateTime zonedStartTime;
```

Comment:
With Java 8, [new date/time related classes](https://docs.oracle.com/javase/8/docs/api/index.html?java/time/LocalDateTime.html) are introduced as successors for the old ``java.util.Date``. There are now "local" date/time classes and there is "zoned" date/time. With "zoned", the time zone is embedded and thus can be converted to anything you want, including UTC. That way, the "old" class is obsolete. See also [this article](https://docs.oracle.com/javase/tutorial/datetime/iso/legacy.html).